### ISSUE TITLE
Add `role="list"` Whenever `list-style: none`

### ISSUE THREAD

**Issue by JamieB-gu** [guardian/dotcom-rendering#5044]
As mentioned in #5041, using `list-style: none` can cause accessibility issues in Safari. @mxdvl describes a solution: add `role="list"` to any unordered list (`ul`) that's had `list-style: none` applied to it.

**Comment by MarSavar** [comment]
We are currently running this plugin in AR: `	jsx-a11y/recommended`  (DCR are using `jsx-a11y`) which produces the following error when trying to add `role="list"` to `ul` tags 🤔 

```
42:2  error  The element ul has an implicit role of list. Defining this explicitly is redundant and should be avoided  jsx-a11y/no-redundant-roles
```
